README.md

```diff
@@ -1081,6 +1082,141 @@ Updating federated Generic repository:
 err = servicesManager.UpdateFederatedRepository().Generic(params)
 ```
 
+#### Creating and Updating Multiple Repositories
```

Comment:
jfrog-client-go is SDK as well so examples should be How to use the new methods you have created for multi repos create/update.